repo/packages/M/msoms/1/package.json

```diff
@@ -0,0 +1,23 @@
+{
+ "packagingVersion": "3.0",
+ "name": "msoms",
+ "version": "1.0.1",
+ "scm": "git@github.com:Microsoft/OMS-docker.git",
+ "maintainer": "omscontainers@microsoft.com",
+ "description": "Microsoft Operations Management Suite(MS OMS) – Container Solution provides real-time analytics for operational data (Performance, Inventory) and monitoring for Docker Containers.",
```

Comment:
yes, merging this one. Will create an issue for the example in the other repo and assign it to you. thanks!